images/Garbage/tune.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Start postgres
+service postgresql start
+
+# Create user
+echo "CREATE USER ${PG_USERNAME} WITH SUPERUSER;" | psql
+
+# Restore schema; /schema.sql should be mounted while execing 
+psql postgres -U ${PG_USERNAME} < /data/dump.sql 
```

Comment:
We should unify the naming of dump.sql and schema.sql.

Right now https://github.com/cmu-db/noisepage-control/blob/49838cf4d7c8f69ed2fb05db74cc8cc8f4a05f2f/replica_daemon/tune.py#L28 is mounting dump.sql to /data, which is correct
But https://github.com/cmu-db/noisepage-control/blob/49838cf4d7c8f69ed2fb05db74cc8cc8f4a05f2f/replica_daemon/base_configs/index_config.json#L11 though it is taking in schema.sql as input